### DISCUSSION TITLE
Sample data

### DISCUSSION THREAD

**Discussion by grd** [grd/FreePDM#40] [Q&A]
Hi, I am looking for sample data that I can analyze. With data I mean parts, bodies, assembly A2P, A3, A4 and drawings. And also other data.

Thanks!

**Comment by grd** [comment]
I am mostly interested in assemblies with multiple levels.

**Comment by grd** [comment]
I found this: 
https://git.vhelio.org/vhelio/vheliotech-freecad (A4 assy)
https://github.com/kisolre/Rosmo_3D

**Reply by DAGABAT** [reply]
Hello! I can help you with some parts, assemblies and drawings.
Let me know how to send it to you!

**Reply by grd** [reply]
Please send it to gvdschoot@gmail.com
Thanks in advance!
